### DISCUSSION TITLE
Mya, houdini USD

### DISCUSSION THREAD

**Discussion by ronnyascencio** [masqu3rad3/tik_manager4#124] [General]
Hello I am just wondering if there a consideration  for the assets workflow when publlish the asset it  published an usd or alembic but whe this is import in to the scene  in maya  it loads as a maya data and not usd data  is this for a reason? 
for example if i  have Asset01 that will be part of the shot 010  on my task of layout i usually import or load the reference Asset01 if is alembic or if the  layout is done in maya reference the maya file, but with usd i should import/load the Asset01 usd file this way always will be updated with the last version published and the the lay out could stored all assets ina usd layout layer  that can be read for animation or lighting  

is this makes sence?

**Comment by masqu3rad3** [comment]
What you are referring is proper usd authoring and before diving into that, it needs to be considered carefully analyzing from a broader perspective.

I believe usd authoring should use the standalone usd api and should have customized extractors and ingestors per dcc.

This is currently not in my short term plans. A proper usd workflow is usually an overkill for individual artists and smaller teams. And bigger studios have their own specific requirement with their dedicated pipe department.

In the future, if I can come up with a 'one size fits all' (or most) solution I can work on it. However it is not my priority at the moment.

I started documentation for custom extractors and ingestors providing necessary bits and pieces. It is possible someone may develop an integration for this before I start on that.
